Дз4/css/style_for_display.css

```diff
@@ -0,0 +1,119 @@
+div {
+  text-align: center;
+  display: inline-block;
+}
+
+.main {
+  display: flex;
```

Comment:
remove display: flex; from this task, we don't have requirements for solutions with flex for this task